aws/ec2.go

```diff
@@ -85,6 +85,20 @@ func nukeAllEc2Instances(session *session.Session, instanceIds []*string) error
 		return errors.WithStackTrace(err)
 	}
 
+	err = svc.WaitUntilInstanceTerminated(&ec2.DescribeInstancesInput{
```

Comment:
Ah, yea, this seems like a good addition. I'd recommend adding equivalent "wait until XXX deleted" to the EBS, ASG, and ELB code.

Reply:
This has been done for everything except classic ELB which doesn't have this functionality

Reply:
If there's no built-in `WaitUntilXXX` method, you can create your own! You just create a retry loop (see [DoWithRetry](https://github.com/gruntwork-io/terratest/blob/06be4e266055d4512a35d3069c43738133eb71b5/util/retry.go#L30)), looking up the ELB by name until it no longer exists or max retries is exceeded.